three.js/src/index-threex.js

```diff
@@ -0,0 +1,21 @@
+import ArMarkerControls from './threex/arjs-markercontrols';
```

Comment:
@MikiDi starting to reviewing the code, to keep a coherence in the code: What the motivation to import MarkerControls as ArMarkerControls from the ARjs namespace, and not creating a THREEx.ArMarkerControl class? Why we can't do the same as for other ARjs classes? I will comment also in the webpack.config.js file about this.

Reply:
When looking at the [equivalent of this on the master branch](https://github.com/AR-js-org/AR.js/blob/f5efd21215a796a2e32b919259e4f5ce839e4b29/three.js/src/threex/threex-armarkercontrols.js#L4), you can see that there already `THREEx.ArMarkerControls` was an alias of `ARjs.MarkerControls` (or the other way around, depending on how you look at it). The above code does exactly the same thing, albeit that the assignment is done through an [import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#importing_defaults) statement, rather than through a variable assignment.
I fully admit that having 2 namespaces, containing exactly the same thing, but under a different class-name, is confusing. Therefore my proposition above to phase out the `THREEx`-namespace. Nonetheless, for this PR, I propose to keep the aliases in here for the sake of reverse compatibility.

Reply:
Yes it is very confusing, note that also other classes are designed in this way as https://github.com/AR-js-org/AR.js/blob/f5efd21215a796a2e32b919259e4f5ce839e4b29/three.js/src/threex/threex-artoolkitprofile.js We should solve this in a future.